### PR TITLE
Premade intercept questions

### PR DESCRIPTION
**Pull request by beamalsky**
## Overview

Connects #104 

This PR creates the following survey questions:

- [x] `age_intercept`: How old are you? 
**Integer**

- [x] `location_intercept`: What's the closest street intersection to where you live? 
**Holding off on this one until we get to  #81**

- [x] `gender_intercept`: What gender do you most identify with? 
**Dropdown: Male, Female, Non-binary, Choose not to answer**

- [x] `income`: What income group does your household fall under? 
**Dropdown: <20k, 20-40k, 40-60k, 60-80k, 80-100k, 100k+**

- [x] `education`: What is your highest formal degree of education? 
**Dropdown: Less than high school diploma, High school diploma, Associate's degree, Bachelor's degree, Graduate/professional degree**

- [x] `race`: Which race or ethnicity best describes you? 
**Dropdown: African-American/Black, Asian, Hispanic or Latino, Native American or Alaska Native, Native Hawaiian or Pacific Islander, Two or more races, White, Some other race**

- [x] `employment`: What is your current employment status?  
**Dropdown: Work full-time, Work part-time, Seeking work, Not seeking work, Student, Homemaker**

- [x] `household_tenure`: How many years have you lived at your current address?
 **Integer**

- [x] `own_or_rent`: Are you a homeowner or a renter? 
**Dropdown**: Renter, homeowner, other

- [x] `transportation`: How did you travel here? 
**Dropdown** ready to drop in these choices

Also to do:
- [x] Write some tests

## Testing Instructions

* If you don't already have it, create the database: `pg_restore -C -j4 --no-owner just-spaces.dump | psql`
* Migrate: `python manage.py migrate`
* Login as testuser (password in the DataMade LastPass under "Just Spaces Staging")
* Create a new Intercept survey
* Add each new element to it. Does the resulting element look like you'd expect?

### CODE REVIEW

**Comment by beamalsky** [comment]
Thought: maybe the labels for these in the dropdown menu should be the questions themselves.

**Comment by jeancochrane** [comment]
@beamalsky Agreed!

**Comment by beamalsky** [comment]
@jeancochrane does ACS offer any categories of transportation, or should be make our own?

**Comment by jeancochrane** [comment]
Sorry I missed this @beamalsky! They do have Means of Transit to Work. Here are the variables: https://github.com/datamade/just-spaces/blob/eda8f4074f27c084aeacff0fceedc5c4d5987096/data/variables.yml#L276-L318

**Comment by beamalsky** [comment]
I've taken a stab at it! 

```
TRANSPORTATION_CHOICES = [
    ('', '-------'),
    ('walked', 'Walked'),
    ('bicycle', 'Bicycle'),
    ('car_truck_van', 'Car, truck, or van'),
    ('motorcycle', 'Motorcycle'),
    ('train', 'Train'),
    ('bus', 'Bus'),
    ('trolley', 'Trolley'),
    ('ferryboat', 'Ferryboat'),
    ('rideshare', 'Rideshare'),
    ('taxicab', 'Taxicab'),
    ('other_means', 'Other'),
]
```

This whole PR is ready for your review @jeancochrane

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
 
-Brief description of what this PR does, and why it is needed.
-
-### Demo
-
-Optional. Screenshots, `curl` examples, etc.
+Connects #XXX
 
-### Notes
+Brief description of what this PR does, and why it is needed.
```
Comment (jeancochrane) [inline .github/pull_request_template.md]: I'm confused why these changes are being displayed in the diff, didn't you already pull them in in #117?

File: fobi_custom/plugins/form_elements/fields/observational/age_observational/forms.py [hunk 2]
```diff
@@ -28,32 +31,13 @@ class AgeObservationalForm(forms.Form, BaseFormFieldPluginForm):
...
-                                     "18-24<br />"
-                                     "25-34<br />"
-                                     "35-44<br />"
-                                     "45-54<br />"
-                                     "55-64<br />"
-                                     "65-74<br />"
-                                     "75+<br /><br />")
+                                     "one of the following levels of detail:<br />"
+                                     "<br /><b>Basic</b><br />"
+                                     + choices_to_help_text(AGE_BASIC_CHOICES)
```
Comment (jeancochrane) [inline fobi_custom/plugins/form_elements/fields/observational/age_observational/forms.py]: Nice refactor :+1:

File: .github/pull_request_template.md (hunk 1 above)
Comment (beamalsky) [reply .github/pull_request_template.md]: I just had to pull in from master!

**Comment by beamalsky** [comment]
I think you're right and that those parenthetical would only be useful for the generic elements. Let me investigate how to suppress those in Fobi.

**Comment by beamalsky** [comment]
On further thought, I think the question type is adding more clutter than it's worth for any question type. I've made that fix here: https://github.com/datamade/django-fobi/pull/4

Thanks for the catch!
